sdk/test/Services/S3/UnitTests/Custom/ResponseMapperTests.cs

```diff
@@ -147,7 +152,7 @@ public void MapPutObjectResponse_AllMappedProperties_WorkCorrectly()
             foreach (var propertyName in putObjectMappings)
             {
                 // Resolve alias to actual property name
-                var resolvedPropertyName = ResolvePropertyName(propertyName);
+                var resolvedPropertyName = ResolvePropertyName(propertyName, "PutObjectResponse");
```

Comment:
nit: we should make `"PutObjectResponse` a variable in this test and then use that here and on this line

```
 .GetProperty("PutObjectResponse")```.

similar in other places